index.html

```diff
@@ -9,6 +9,6 @@
   <script src="main.js"></script>
 </head>
 <body>
-  <h1>Master</h1>
+  <h1 style="font-size: 300%">B1</h1>
```

Comment:
No usar estilos inline, crear una clase en estilos.cc
```suggestion
  <h1 class="title">B1</h1>
```